### ISSUE TITLE
pip install daal4py==2021.7.1 fails on Python 3.9

### ISSUE THREAD

**Issue by 2-5** [intel/scikit-learn-intelex#1066]
**Describe the bug**
`pip install daal4py==2021.7.1` fails on Python 3.9 because it seems this version requires `Python ==3.8.*`

**To Reproduce**
1. create Python 3.9 venv: `python3.9 -m venv venv`
2. activate it
3. `pip install daal4py==2021.7.1`
4. fails with error:

```log
ERROR: Ignored the following versions that require a different python version: 2021.6.0 Requires-Python ==3.10.*; 2021.7.1 Requires-Python ==3.8.*
ERROR: Could not find a version that satisfies the requirement daal4py==2021.7.1 (from versions: 2021.1, 2021.2.0, 2021.2.1, 2021.2.2, 2021.2.3, 2021.3.0, 2021.4.0, 2021.5.0, 2021.5.3, 2021.5.4, 2021.6.2, 2021.6.3, 2021.7.0)
ERROR: No matching distribution found for daal4py==2021.7.1
```
**Output/Screenshots**
See PyPI sidebar which says Python must be exactly 3.8:

https://pypi.org/project/daal4py/2021.7.1/

![2022-11-22_145146](https://user-images.githubusercontent.com/3753822/203318772-9cbed6df-5689-4f46-8316-01bc7a7f8556.png)

**Environment:**
 - OS: Windows 10
 - Version: 2021.7.1

**Comment by Alexsandruss** [comment]
Hello, I addressed this problem to maintainer of pypi packages.

**Comment by tomashek** [comment]
@2-5 , We confirmed that the package does not install from the index. Further investigation revealed that the package itself is available on the index, and the "Requires-Python:" is correctly set to "==3.9.*" and not 3.8. I reported this to PyPI because it appears to be a problem in the index, not the packages. See https://github.com/pypa/packaging-problems/issues/631.

A verified workaround is to first install daal==2021.7.1 and numpy, then [download the wheel](https://files.pythonhosted.org/packages/09/e8/eba1b4b0fd16412de0c7d4496d89a28e389db357ad7b436e8bf1c6eda7c5/daal4py-2021.7.1-py39-none-win_amd64.whl) and install it from the local reference with "pip install daal4py-2021.7.1-py39-none-win_amd64.whl"

**Comment by 2-5** [comment]
If you access PyPI through the JSON api for this version you can see that all packages have the `==3.8.*` requirements. Maybe the uploader script has an issue. 

https://pypi.org/pypi/daal4py/2021.7.1/json

Simplified:

```json
{
  "info": {
    "name": "daal4py",
    "package_url": "https://pypi.org/project/daal4py/",
    "platform": null,
    "project_url": "https://pypi.org/project/daal4py/",
    "release_url": "https://pypi.org/project/daal4py/2021.7.1/",
    "requires_dist": ["daal (==2021.7.1)", "numpy (>=1.15)"],
    "requires_python": "==3.8.*",
    "version": "2021.7.1"
  },
  "last_serial": 15695830,
  "urls": [
    {
      "filename": "daal4py-2021.7.1-py310-none-macosx_10_15_x86_64.macosx_11_0_x86_64.whl",
      "python_version": "py310",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py310-none-manylinux1_x86_64.whl",
      "python_version": "py310",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py310-none-win_amd64.whl",
      "python_version": "py310",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py37-none-macosx_10_15_x86_64.macosx_11_0_x86_64.whl",
      "python_version": "py37",
      "requires_python": "==3.8.*",
    },
    {
      "filename": "daal4py-2021.7.1-py37-none-manylinux1_x86_64.whl",
      "python_version": "py37",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py37-none-win_amd64.whl",
      "python_version": "py37",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py38-none-macosx_10_15_x86_64.macosx_11_0_x86_64.whl",
      "python_version": "py38",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py38-none-manylinux1_x86_64.whl",
      "python_version": "py38",
      "requires_python": "==3.8.*",
    },
    {
      "filename": "daal4py-2021.7.1-py38-none-win_amd64.whl",
      "python_version": "py38",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py39-none-macosx_10_15_x86_64.macosx_11_0_x86_64.whl",
      "python_version": "py39",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py39-none-manylinux1_x86_64.whl",
      "python_version": "py39",
      "requires_python": "==3.8.*"
    },
    {
      "filename": "daal4py-2021.7.1-py39-none-win_amd64.whl",
      "python_version": "py39",
      "requires_python": "==3.8.*"
    }
  ],
  "vulnerabilities": []
}
```

**Comment by napetrov** [comment]
Fixed with 2023.0.0 release and dependency set to >=3.7 python for all packages on pypi

**Comment by 2-5** [comment]
@napetrov It seems the 2023.0.0 release has no Windows wheels:

https://pypi.org/project/daal4py/2023.0.0/#files

Is that by design?

**Comment by napetrov** [comment]
@2-5 No, they intend to be there.  But guess this would be available only after NY once people would be back to push this.
Also there would be patch package with bunch of fixes including better 1.2 scikit compatibility
